ompi/mca/osc/rdma/osc_rdma_accumulate.c

```diff
@@ -314,7 +314,7 @@ static inline int ompi_osc_rdma_gacc_amo (ompi_osc_rdma_module_t *module, ompi_o
     if (NULL != result_convertor) {
         /* result buffer is not necessarily contiguous. use the opal datatype engine to
          * copy the data over in this case */
-        struct iovec iov = {.iov_base = result, .iov_len = request->len};
+        struct iovec iov = {.iov_base = result_start, .iov_len = request->len};
```

Comment:
If `result != NULL` then `result_start` is `NULL` (per https://github.com/open-mpi/ompi/pull/10634/files#diff-e4bd30eae59fa8aabf9ee87dcb523aa60a292e7c09d1e84e014c1ae7e13e1e3cR281). `result_start` should always point to the start of the `result` buffer.

Reply:
Thanks for the review, updated the PR with the changes, there was also a memory leak here, https://github.com/open-mpi/ompi/pull/10634/files#diff-e4bd30eae59fa8aabf9ee87dcb523aa60a292e7c09d1e84e014c1ae7e13e1e3cR311 which is fixed.